<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendFilePropertise</class>
 <widget class="QWidget" name="SendFilePropertise">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Send Propertise</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Project:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="projectCbox">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::InsertAtTop</enum>
          </property>
          <item>
           <property name="text">
            <string>none</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editProjectBtn">
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>ID:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="projectIdCbox">
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>none</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Zone:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="projectZoneCbox">
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>none</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" rowspan="2" colspan="2">
       <widget class="QLabel" name="projectPathLb">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
